frontend/model/notifications/templates.js

```diff
@@ -145,17 +145,33 @@ export default ({
     }
   },
   NEW_PROPOSAL (data: { groupID: string, creatorID: string, proposalHash: string, subtype: NewProposalType }) {
-    const args = {
-      name: `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${data.creatorID}`,
-      ...LTags('strong')
-    }
+    const isCreator = data.creatorID === sbp('state/vuex/getters').ourIdentityContractId // notification message is different for creator and non-creator
+    const args = isCreator
+      ? null
+      : {
+          name: `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${data.creatorID}`,
+          ...LTags('strong')
+        }
```

Comment:
It's not like `args` is used if `isCreator` is `true`, so this `?:` isn't needed here, these changes can be reverted.